### PR TITLE
Re-worked on home page banner link alignment fix

### LINKED ISSUE #7794: `What's new in 2.7` and `Preferences` hyperlinks have moved to the next line in Home Page
**On 2.7-head commit id: `65089ee0`**

Screen Shot 2022-12-21 at 8 21 12 PM

**On 2.6.9**
Screen Shot 2022-12-21 at 8 24 08 PM

Can we have them on the same line, if needed a little closer than 2.6.9 ?

### PR DESCRIPTION
**Pull request by bisht-richa**
### Summary
Fixes #7794

Re-worked on home page banner link alignment fix as the current fix was breaking the banner in cluster > auth > config page

### Issue
**Before**

**c/local/auth/config page** 

Screenshot 2023-01-16 at 17 14 28

**Home page banner**
Screenshot 2023-01-16 at 17 51 58

### After fix
**c/local/auth/config**
Screenshot 2023-01-16 at 17 50 02

**Home page banner**
Screenshot 2023-01-16 at 17 49 49

### CODE REVIEW

**Review by torchiaf — DISMISSED** [review]
LGTM

**Review by nwmac — CHANGES_REQUESTED** [review]
@bisht-richa Can you check - I see the GH auth banner now not looking right.

was:
image

with your PR, is:
image

**Comment by nwmac** [comment]
@bisht-richa The homepage banners appear to have been already fixed - just the auth banner looks to be broken on master.

File: shell/pages/home.vue [hunk 1]
```diff
@@ -301,7 +301,7 @@ export default {
             data-testid="changelog-banner"
             color="info whats-new"
           >
-            <div class="message">
+            <div>
```
Comment (nwmac) [inline shell/pages/home.vue:304]: What do the changes in this file do? These banners seem to fine now in the master branch - are these changes needed?

**Review by nwmac — DISMISSED** [review]
@bisht-richa Neither of these changes appears to be needed any longer - checking master, it seems fine there - can you check and close out if so.

File: shell/pages/home.vue (hunk 1 above)
Comment (bisht-richa) [reply shell/pages/home.vue:304]: Yes, I added this class in the first PR to fix the home banner and did some changes in the banner component. But after new fixes we don't need these extra classes.

**Comment by torchiaf** [comment]
> @bisht-richa Neither of these changes appears to be needed any longer - checking master, it seems fine there - can you check and close out if so.

Thanks Neil, I can confirm that the `Authentication Providers` section still requires the fix. Below pics is taken from the master:
image

**Review by torchiaf — CHANGES_REQUESTED** [review]
Thanks @bisht-richa please verify that your commits are still required in the master `HEAD`, otherwise do a `reset`. See my previous commit.

**Comment by bisht-richa** [comment]
> 

Yes it is valid.

**Comment by torchiaf** [comment]
> > 
> 
> @torchiaf Yes, it is required.

In this case LGTM.

**Review by torchiaf — APPROVED** [review]
LGTM

**Pull request merged**